Runtime/Client/LootLockerHTTPClient.cs

```diff
@@ -76,6 +75,10 @@ public class LootLockerHTTPClientConfiguration
          * The maximum number of requests allowed to be in progress at the same time
          */
         public int MaxOngoingRequests = 50;
+        /*
+         * The maximum size of the request queue before new requests are rejected
+         */
+        public int MaxQueueSize = 5000;
```

Comment:
Minor, either set this one to 1000 or the one at line 101 to 5000 to avoid confusion if we need to debug it.